packages/react-components/react-migration-v0-v9/src/components/Spinner/SpinnerMigration.mixins.ts

```diff
@@ -0,0 +1,15 @@
+import { GriffelStyle, tokens } from '@fluentui/react-components';
+
+const inline = (): GriffelStyle => ({ display: 'inline-flex' });
+
+const v0SpinnerLabelStyle = (): GriffelStyle => ({
+  '& .fui-Label': {
```

Comment:
Can't we use `labelClassNames.root` there?

Reply:
Done